packages/salesforcedx-vscode-apex/OSSREADME.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
@ntotten could you verify that I'm not missing something here ? The original code is on https://github.com/Microsoft/vscode/blob/9d611d4dfd5a4a101b5201b8c9e21af97f06e7a7/extensions/typescript/src/typescriptMain.ts#L186

Reply:
Seems good to me. AFAIK, we just need to include a copy of the license.